.github/dependabot.yml

```
@@ -0,0 +1,30 @@
+version: 2
```

nit:
```suggestion
---
version: 2
```
+ `\n` missing at the end of the file